core/domain/classifier_services_test.py

```diff
@@ -253,3 +253,58 @@ def test_update_of_classifier_training_jobs(self):
             classifier_services.get_classifier_training_job_by_id(job_id))
         self.assertEqual(classifier_training_job.exp_id, exp_id)
         self.assertEqual(classifier_training_job.status, test_status)
+
+    def test_retrieval_of_classifier_from_exploration_attributes(self):
+        """Test the get_classifier_from_exploration_attributes method."""
+
+        with self.assertRaisesRegexp(Exception, (
+            'Entity for class ClassifierDataModel with id '
+            'fake_id not found')):
+            classifier_services.get_classifier_by_id('fake_id')
+
+        exp_id = u'1'
+        state_name = 'Home'
```

Comment:
@anmolshkl . Again, not required. The state name does not need to be compulsorily unicode. It needs to work if the state name is ascii as well.